actors/abi/sector.go

```diff
@@ -95,53 +95,35 @@ const (
 // Metadata about a seal proof type.
 type SealProofInfo struct {
 	SectorSize                 SectorSize
-	WindowPoStPartitionSectors uint64
```

Comment:
Is this not a limitation of the proof type?

Reply:
I'm pretty sure it's policy, in that it was chosen based on security analysis, and new analysis could change it. It would have to be matched in the proofs library, but it's a feasible upgrade.